server

```diff
@@ -61,7 +61,8 @@ Not enough variations
 END
             return 0;
         fi
-        ./fractal_gen -w $width -h $height $fracargs > /tmp/fractal$$.ppm;
+        ./fractal_gen -w $width -h $height $fracargs
+        mv out000.ppm /tmp/fractal$$.ppm
```

Comment:
What if two people are requesting a fractal simultaneously? There would be two instances of cfractal trying to write to the same file.